powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/exception/DefaultExceptionResolver.java

```diff
@@ -56,6 +72,7 @@
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     public @ResponseBody ErrorResponse handleDefaultException(Throwable t) {
         logger.error("Error occurred in Next Step server", t);
+        audit.error("Error occurred in Next Step server", AUDIT_DETAIL_UNEXPECTED_ERROR, t);
```

Comment:
I am not sure if we want to audit all exceptions - there are already stored in logger appender and do not represent the business logic situation as much as application errors. The purpose of audit log is to store business audit (what the user was doing in the system), not technical logging.